integration_test/common.js

```diff
@@ -307,11 +324,88 @@ module.exports = {
     return [app1, app2];
   },
 
+  async addFriendToNewClosedGroup(app, app2) {
+    await app.client
+      .element(ConversationPage.closedGroupNameTextarea)
+      .setValue(this.VALID_CLOSED_GROUP_NAME1);
```

Comment:
You have to call the setValueWrapper here and for all your setValue calls otherwise it will break on darwin